eo-runtime/src/test/eo/org/eolang/txt/sprintf-tests.eo

```diff
@@ -43,3 +43,8 @@
       "Jeff"
       "X"
     $.equal-to "Привет 3.14 Jeff X!"
+
+[] > prints-bytes-string
+  assert-that > @
+    sprintf "%c" (56.as-bytes.slice 7 1)
```

Comment:
@Graur are we have difference between `%s` and `%c` in EO. Since we don't have a Char type, it probably makes sense to use the `%s` here.

Reply:
@EugeneDar please, read the related issue: #1049